### PR TITLE
Add Dutch translation

### PR DESCRIPTION
**Pull request by SimonTil**
Hereby a small contribution from a Prusa-fanboy

### CODE REVIEW

File: src/locales/source/nl.json [hunk 1]
```diff
@@ -0,0 +1,259 @@
...
+  "btn.extrude": "Extrudeer",
+  "btn.login": "Login",
+  "btn.no": "Nee",
+  "btn.pause-pt": "Pauzeer print",
+  "btn.resume-pt": "Hervat print",
+  "btn.retract": "Retract",
+  "btn.save-chgs": "Wijzigingen opslaan",
+  "btn.set": "Instellen",
+  "btn.sla-refill": "Resin bijvullen",
+  "btn.sla-refilled": "Resin bijgevuld,
```
Comment (Lowentwickler) [inline src/locales/source/nl.json:23]: Missing quote.

File: src/locales/update_translations.py [hunk 2]
```diff
@@ -11,7 +11,8 @@
     ("es","es_ES"),
     ("fr","fr_FR"),
     ("it","it_IT"),
-    ("kr","kr_KR")
+    ("kr","kr_KR"),
+    ("nl","nl-NL")
```
Comment (Lowentwickler) [inline src/locales/update_translations.py:15]: Please, use `nl_NL` instead

**Review by Lowentwickler — CHANGES_REQUESTED** [review]

**Comment by Lowentwickler** [comment]
Thank you for your contribution :)

**Comment by Lowentwickler** [comment]
Please, also add `nl` to [this file](https://github.com/prusa3d/Prusa-Link-Web/blob/f9acf204d2957fcace6419ccb3f98aae2158c1e7/tools/preprocessing/extract-words.js#L13)

**Comment by SimonTil** [comment]
I am not really familiar with GitHub, but I changed the files to my branch. Should be fixed now?

**Comment by Lowentwickler** [comment]
Yes, it looks good now. Thank you!

**Review by ondratu — APPROVED** [review]

**Comment by SimonTil** [comment]
Don't know what that notification means? Merging is blocked, but why? Did I do something wrong?

**Pull request merged**